docs/installation.md

```diff
@@ -115,10 +115,16 @@ Once you have Docker installed, simply create the config file (e.g. `config.json
 
 **1.1. Clone the git repository**
 
+Linux
```

Comment:
Linux / MacOS ?

Reply:
True. I update tomorrow with the test of WSL.